RecoHGCal/TICL/python/ticl_iterations.py

```diff
@@ -54,7 +54,14 @@ def TICL_iterations_withReco(process):
       missing_layers = 2,
       min_clusters_per_ntuplet = 15,
       min_cos_theta = 0.94, # ~20 degrees
-      min_cos_pointing = 0.7
+      min_cos_pointing = 0.7,
+      eid_graph_path = cms.string("RecoHGCal/TICL/data/tf_models/energy_id_v0.pb"),
```

Comment:
These are all the default values for the newly added parameters, that you don't need to overwrite here in these explicit configs (the same as you do for ```TrackstersMIP```)

Reply:
✔︎